google/resumable_media/_download.py

```diff
@@ -162,10 +185,15 @@ def _process_response(self, response):
         self._finished = True
         _helpers.require_status_code(
             response, _ACCEPTABLE_STATUS_CODES, self._get_status_code)
+        if self._stream is not None:
+            self._write_to_stream(response)
```

Comment:
@tseaver Sorry I missed your comment when I made mine.

I am (for now) just punting on figuring out what the sans-I/O approach should be. I will be following this up with modifying `requests.Download.consume()` to do the HTTP chunking in the "I am doing I/O now" section.